src/helpers/timeConversions.ts

```diff
@@ -41,3 +41,22 @@ const isPlural = (num: number, timeUnit: string) => {
 export const round = (num: number, digits: number): number => {
     return +(num).toFixed(digits) 
 };
+
+
+export enum TimeUnit {
+    SECONDS = 1,
+    MINUTES = 60,
+    HOURS = 3600,
+    DAYS = 86400,
+    WEEKS = 604800,
+    MONTHS = 2628000,
+    YEARS = 31536000
+  }
+
+export const convertTime = (value: number, from: TimeUnit, to: TimeUnit): number => {
+    const timeInSeconds = value * from;
+    const convertedTime = timeInSeconds / to;
+    return convertedTime;
+}
+
+export const daysOfWeek = ["MONDAY", "TUESDAY", "WEDNESDAY", "THURSDAY", "FRIDAY", "SATURDAY", "SUNDAY"];
```

Comment:
Minor: constants are usually named using uppercase for everything, for example: `DAYS_OF_WEEK`. This allows other developers to know that value is not intended to be changed through the program execution. In the same way, `TimeUnit` (line 46) might also be changed to follow this convention.